src/__test__/__reducers__/userReducer.test.js

```diff
@@ -0,0 +1,29 @@
+import { userActionsTypes } from '../../actions-types';
+import userReducer from '../../reducers/userReducer';
+import initialState from '../../store/initialStates/userInitialState';
+
+describe('User reducer', () => {
+  it('login success', () => {
+    const payload = {
```

Comment:
It will be better if we have this payload defined in the mock user file to have proper and clean code.